tests/unit_tests/test_csv_splitter_unit.py

```diff
@@ -21,52 +22,37 @@ def temp_file() -> StringGeneratorType:
 temp_csv_file = create_temp_file_fixture(file_extension="csv")
 temp_txt_file = create_temp_file_fixture(file_extension="txt")
 
-
-@pytest.fixture
-def create_files_from_chunks():
-    create_files(chunks=CHUNKS, output_file_name="out")
-    yield
-    os.remove("out_1.csv")
-    os.remove("out_2.csv")
+valid_row_limits = random.randint(2, 7)
+invalid_row_limits = random.randint(8, 20)
 
 
 def test_validate_file_length_success(temp_csv_file):
-    assert validate_file_length(input_file=temp_csv_file, row_limit=3) is True
+    assert validate_file_length(input_file_path=temp_csv_file, row_limit=valid_row_limits) is True
 
 
 def test_validate_file_length_should_fail_row_limit_too_high(temp_csv_file):
-    assert validate_file_length(input_file=temp_csv_file, row_limit=8) is False
+    assert validate_file_length(input_file_path=temp_csv_file, row_limit=invalid_row_limits) is False
 
 
 def test_is_input_valid_success(temp_csv_file):
-    assert is_input_valid(input_file=temp_csv_file, row_limit=5) is True
+    assert is_input_valid(input_file_path=temp_csv_file, row_limit=valid_row_limits) is True
 
 
 def test_is_input_valid_should_fail_file_not_exist():
-    assert is_input_valid(input_file="non_existent_file.csv", row_limit=3) is False
+    assert is_input_valid(input_file_path="non_existent_file.csv", row_limit=valid_row_limits) is False
 
 
 def test_is_input_valid_should_fail_not_csv(temp_txt_file):
-    assert is_input_valid(input_file=temp_txt_file, row_limit=3) is False
+    assert is_input_valid(input_file_path=temp_txt_file, row_limit=valid_row_limits) is False
 
 
 def test_get_file_chunks(temp_csv_file):
-    chunks = get_file_chunks(temp_csv_file, 4)
+    row_limit_for_two_chunks = 4
```

Comment:
Yes, but why is this the row limit? how did you get to the number 4? is it the `len` of something? can it be calculated in the code?